index-test.js

```diff
@@ -39,33 +39,19 @@ test("form submits expected result", () => {
 
 */
 
-setTimeout(() => {
-
-  test("deleteItem() function should remove an item from the list", () => {
-    const deleteBtn = document.querySelectorAll('button');
-    const movieDiv = document.querySelectorAll('.movie-name');
-    const movietitle = document.querySelectorAll('.movie-title');
-    if(this.item === undefined) {return} //This stops the .click() from returning an error message. 
-    deleteBtn[0].click(); //When the first delete button is clicked...
-    const result = movieDiv.includes(movietitle[0])
-    const expected = false
-    equal (result, expected);
-    deleteBtn[0].click();
-  })
-  }, 5000);
-  
-
-setTimeout(() => { //Allows time for the elements to appear on the page as they're not present onload.
-test("deleteItem() function should remove an item from the list", () => {
-  const movieDiv = document.querySelectorAll('.movie-name') //Outer container
-  const movieTitle = document.querySelectorAll('.movie-title') //Input value
-  const deleteBtn = document.querySelectorAll('button') //delete button
-  if(this.item === undefined) {return} //This stops the .click() from returning an error message. 
-  let result = deleteBtn[0].click(); //When the first delete button is clicked...
-  const expected = deleteItem(movieDiv.contains(movieTitle[0]) == false); //...the first input will no longer be contained in the div.
-  // const expected = false;
-  equal (result, expected); //When result takes place, expected is what we should see.
-  })
-  }, 5000);
-
-//Remove the delete button's class name. (Added in for test purposes)
+
+
+test("Delete button removes items from the list", () => {
+document.querySelector('button#new-list').click(); //Click the TV button to generate new list
+const input = document.querySelector('input');
+input.value = "test"; //Add input value.
+document.querySelector('button#add-title').click(); //Click the 'plus' button to submit input.
+const todo = document.querySelector('.movie-name');
+todo.querySelector('button.delete').click(); //Click the delete button in the movie-name div to delete the input.
+const todoAfter = document.querySelector('.movie-name');
+equal(todoAfter, null, 'Item has been deleted.') //After the input has been deleted, the querySelecting movie-name should show up as null bc the item is deleted.
+todo.remove() //Clear the screen.
+document.querySelector('.list-container').remove();
```

Comment:
Well remembered ro reset the page!!

Reply:
Good addition with the .remove() to clearing the screen as a restart, great work 🎉